android/src/main/java/com/azendoo/reactnativesnackbar/SnackbarModule.java

```diff
@@ -20,12 +22,15 @@
 
 public class SnackbarModule extends ReactContextBaseJavaModule{
 
+    Context context;
+
     private static final String REACT_NAME = "RNSnackbar";
 
     private List<Snackbar> mActiveSnackbars = new ArrayList<>();
 
     public SnackbarModule(ReactApplicationContext reactContext) {
         super(reactContext);
+        this.context = reactContext.getApplicationContext();
```

Comment:
Instead of saving a member var that could be nullified when the app goes away, let's just use a local `Context context 
 = getReactApplicationContext().getApplicationContext()` below.

Reply:
Sorry, I don't now much about android. so I can't understand where should I put this line.